README.md

```diff
@@ -52,24 +43,43 @@
 
 - Deploy the chaos-exporter.yaml 
 
-- From a cluster node, execute `curl <exporter-service-ip>:8080/metrics` 
+- From a cluster node, execute `curl <exporter-service-ip>:8080/metrics`  or `curl <pod-ip>:8080.metrics`
```

Comment:
we need the service as well, for exporting the metrics

Reply:
Yes, the first one is the service ip, and other is the pod ip

Reply:
I don't think we need to tell the user to curl on the pod IP as well

Reply:
yes

Reply:
Will change it to deployment.